tensorboard/plugins/graph/tf_graph_common/render.ts

```diff
@@ -334,15 +334,33 @@ export class RenderGraphInfo {
    */
   getNearestVisibleAncestor(name: string): string {
     let path = getHierarchicalPath(name);
-    for (let i = 0; i < path.length; i++) {
-      let nodeName = path[i];
+    let i = 0;
+    let renderNode : RenderNodeInfo = null;
+    // Fallthrough. If everything was expanded return the node.
```

Comment:
Yes. Technically the default case is still the clicked node name. It could occur if every node in the hierarchical paths were thought to be expanded. While I think that shouldn't happen, I guess it was originally there just as a safety check.